actionmailer/lib/rails/generators/mailer/USAGE

```diff
@@ -12,6 +12,5 @@ Example:
 
     creates a Notifications mailer class, views, and test:
         Mailer:     app/mailers/notifications_mailer.rb
-        Views:      app/views/notifications/signup.text.erb [...]
-        Test:       test/mailers/notifications_test.rb
-
+        Views:      app/views/notifications_mailer/signup.text.erb [...]
+        Test:       test/mailers/notifications_mailer_test.rb
```

Comment:
@carlosantoniodasilva Thank you for pointing that out. It's my mistake. I updated.